sdk/core/azure-core/inc/azure/core/http/http.hpp

```diff
@@ -603,29 +610,30 @@ namespace Azure { namespace Core { namespace Http {
     // ===== Methods used to build HTTP response =====
 
     /**
-     * @brief Add header to the HTTP response.
+     * @brief Add header to the HTTP response. Returns 0 if header is successfully added, otherwise
```

Comment:
The details about `return` should be in the `@return` tag, and keep the brief brief.